### ISSUE TITLE
Example for the APIs

### ISSUE THREAD

**Issue by willpowerforever** [aquto/cordova-plugin-vpn#2]
Hi,

Could you create an example about how to use the API? I am new to the JS technology and trying to figure out how to use it. It will be great if you can provide an example.

Thanks!
William

**Comment by MickdeGraaf** [comment]
I use this code to try to connect but I get an unknown error because I don't know what to fill in in the certificate and certificate password field.


```javascript

window.plugins.VPNManager.enable(
	    										function(result){
		    										console.log('connected');
	    										},
	    										function(error){
		    										console.log(error);
	    										},
	    										{
		    										"vpnPassword": "password",
		    										"vpnUsername": "username",
		    										"vpnHost": "nrt-a01.vpn.asia",
		    										"certificate": "cert",
		    										"certificatePassword": "certpass"
	    										}
	    										
	    
    											);	


```
What do I need to fill in in those fields and do you maybe have a code example?

Thanks,

Mick de Graaf

**Comment by electricfeel1979** [comment]
Good day, yes an example would be great. I am a beginner with using cordova as well.

**Comment by jprichardson** [comment]
Would love an example as well :)

**Comment by Misiu** [comment]
Yes, example of this would be really nice, especially for newbies.

**Comment by caron7** [comment]
hello, would you please tell me how to connect a vpn server whitch uses ipsec xauth psk?
thanks!

**Comment by xxxlucaxxx** [comment]
Hi, would you be able to post some example how to establish the VPN connection?
Unfortunate, it is very hard to follow up how you are establishing the connection.

Thanks!

**Comment by MickdeGraaf** [comment]
I don't expect we'll be getting an answer any time soon.

**Comment by SlavaVolkov** [comment]
Hay everyone guys! Did somebody estabish vpn connection with this plugin?Sorry for my English.
I have an error when I build my app with this plugin in NetworkExtension framework:
![screenshot_2](https://cloud.githubusercontent.com/assets/10748073/13058227/84c59e68-d42a-11e5-8aad-d842da81eccb.jpg)

**Comment by CELIKEM** [comment]
It's not working so he's not answering any question.

**Comment by maobonet** [comment]
This plugin only supports ikev2-cert-eap

**Comment by Khsed4** [comment]
@willpowerforever @CELIKEM @MickdeGraaf @SlavaVolkov  Hey guys, I need to implement a VPN, I was wondering if you solved the problem anyway, I don't care which programming language and which tools just could you please make me inform how I can create an L2TP VPN?

**Comment by McSam94** [comment]
Hi guys, any update on this matter? is this library still usable?

**Comment by akkariyassine** [comment]
did you found a solution ?

**Comment by redbrain** [comment]
@aquto example usage would be highly appreciated
